src/functionalTest/resources/data/opa-results/opaResults.json

```diff
@@ -0,0 +1,103 @@
+{
+  "document": {
+    "publicationDate": "2023-09-13T23:30:52.123Z",
+    "documentName": "This is a document name",
+    "version": "1.0"
+  },
+  "venue": {
+    "venueName": "This is the venue name",
+    "venueContact": {
+      "venueEmail": "a@b.com",
+      "venueTelephone": "+44 1234 1234 1234"
+    },
+    "venueAddress": {
+      "line": [
+        "Address Line 1",
+        "Address Line 2"
+      ],
+      "town": "Town",
+      "county": "County",
+      "postCode": "AA1 1AA"
+    }
+  },
+  "courtLists": [
+    {
+      "courtHouse": {
+        "courtRoom": [
+          {
+            "session": [
+              {
+                "sittings": [
+                  {
+                    "hearing": [
+                      {
+                        "case": [
+                          {
+                            "caseUrn": "URN1234"
+                          }
+                        ],
+                        "party": [
+                          {
+                            "partyRole": "DEFENDANT",
+                            "individualDetails": {
+                              "individualForenames": "Forename",
+                              "individualMiddleName": "MiddleName",
+                              "individualSurname": "Surname"
+                            },
+                            "offence": [
+                              {
+                                "offenceTitle": "Offence title 1",
+                                "offenceSection": "Offence section 1",
+                                "decision": {
+                                  "decisionDate": "",
```

Comment:
Can these two fields be populated with dummy data

Reply:
Same for other list below

Reply:
Must have been day dreaming on Monday. The test files have incorrect fields/format. I have replaced the test files